const userInfo = async() => {
  try {
    
  } catch (error) {
    console.log(error);
  }
}